modules/nextflow/src/test/groovy/nextflow/container/ShifterBuilderTest.groovy

```diff
@@ -57,26 +57,44 @@ class ShifterBuilderTest extends Specification {
         when:
         def cli = new ShifterBuilder('ubuntu:14').build().getRunCommand()
         then:
-        cli ==  '''
+        cli.stripIndent().trim() ==  '''
                 shifterimg pull ubuntu:14
```

Comment:
This is not a good idea. The command string should be produced without indentation instead of removing it to make the test pass! 😬

Reply:
Of course you are right :smile:
The truth is I was just having trouble with testing what indentation comes out of the getRunCommand, and I thought that in this case indentation was not really an issue.

I will submit a new change shortly :)